### DISCUSSION TITLE
Assistance Needed with ADFS Server Name Change

### DISCUSSION THREAD

**Discussion by MilSN** [neos-sdi/adfsmfa#356] [Q&A] [answered]
Dear @redhook62 

I hope this message finds you well.

I recently renamed my ADFS primary and secondary machine names; however, I am still seeing the old server name reflected in the services status. Additionally, the secondary server is not appearing as expected.
![ADFSMFA](https://github.com/user-attachments/assets/72f260df-16f0-4379-a78e-a64f3fd91955)

Could you please provide guidance on how to resolve these issues?

Thank you for your assistance.

Best regards,

**Comment by redhook62** [comment, marked as answer]
Hi,

try to use these cmdlets 

Unregister-MFAComputer  (For old names)
and
Register-MFAComputer (For new names)

Additionally, you can export your configuration and remove / computer in the xml file
Export-MFASystemConfiguration 
and
Import-MFASystemConfiguration 

regards
